client/packages/system/src/InventoryAdjustmentReason/Components/InventoryAdjustmentReasonSearchInput.tsx

```diff
@@ -44,13 +46,14 @@ export const InventoryAdjustmentReasonSearchInput: FC<
   };
   const reasons = (data?.nodes ?? []).filter(reasonFilter);
 
-  const isRequired = reasons.length !== 0;
+  const reasonsAvailable = reasons.length !== 0;
+  const isRequired = reasonsAvailable && initialStocktake != true;
```

Comment:
Reasons may or may not be configured (in OG).
If reasons exist, and the stocktake is not initial, an adjustment reason is a required field for any changed quantities

Reply:
you can use the 'not' operate to check bools
```suggestion
  const isRequired = reasonsAvailable && !initialStocktake;
```